spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/export/influx/InfluxProperties.java

```diff
@@ -33,7 +35,7 @@
 public class InfluxProperties extends StepRegistryProperties {
 
 	/**
-	 * Tag that will be mapped to "host" when shipping metrics to Influx.
+	 * Destination to write metrics when using an InfluxDB 1.x database.
```

Comment:
The previous description was just wrong and some version of the updated description should probably be backported.

Reply:
Thanks Tommy. I've created: https://github.com/spring-projects/spring-boot/issues/25723